src/services/auth/index.tsx

```diff
@@ -1,57 +1,69 @@
 import { IS_MOCK } from "../../config";
-import { Policy } from "../../types";
```

Comment:
We'll need to add this back for now

Reply:
as in the whole Policy being returned from `/confirm`?

Reply:
We'll use `/auth` first, and later refactor